packages/bitcore-node/src/modules/ethereum/models/transaction.ts

```diff
@@ -18,14 +18,28 @@ import { BaseTransaction } from '../../../models/baseTransaction';
 import { valueOrDefault } from '../../../utils/check';
 import { InvoiceAbi } from '../abi/invoice';
 
-const Erc20Decoder = require('abi-decoder');
-Erc20Decoder.addABI(ERC20Abi);
+function requireUncached(module) {
```

Comment:
😬

Reply:
The module itself caches all the ABIs, so you have to have a clean one :(